docker-compose.yml

```diff
@@ -332,8 +387,7 @@ services:
       n2htoleaf2:
     # L2 VXLAN - VLAN10 stretched with VNI10 to leaf2 from leaf1 (ping 10.10.10.3)
     command: |
-      sh -x -c 'sleep 30 && \
```

Comment:
why sleep is removed ?

Reply:
I see that BGP sessions are established quickly, we dont need any delay.